spec/project_spec.rb

```diff
@@ -3689,4 +3823,120 @@ def callback(_args); end
       end
     end
   end
+
+  describe '#decide_all' do
```

Comment:
These tests should be changed. We cannot see the impact of ENABLED_FLAGS_ONLY option since all flags are enabled here  :)

Reply:
Its already being checked in the very next test
https://github.com/optimizely/ruby-sdk/blob/7f6d985512918a089d32bc7998db484127ebbd9c/spec/project_spec.rb#L3856

Reply:
My bad. I didn't see checking the count of enabled flags.